js/classes/AuthorsMenu.js

```diff
@@ -0,0 +1,75 @@
+import { ApiClient } from '../helpers/ApiClient.js';
+import { BlogPost } from '../components/BlogPost.js';
+
+export class AuthorsMenu {
+  constructor (authorsList, { subscribe, publish }) {
+    this.authorsList = authorsList;
+    this.vertical = this.authorsList.classList.contains('posts-search__authors-list--vertical');
+    this.subscribe = subscribe;
+    this.publish = publish;
+    this.handleLoad();
+  }
+
+  postDescriptionContainer = document.querySelector('.posts-search__post');
```

Comment:
This is getter of a DOM element. So, this method should be called as `getPostDescription`. `Container` name part looks redundant.